src/shared/Textarea/Textarea.styled.js

```diff
@@ -0,0 +1,47 @@
+import { TextareaAutosize } from '@mui/base/TextareaAutosize';
+
+import styled from '@emotion/styled';
+
+export const TextareaStyled = styled(TextareaAutosize)`
+  width: 100%;
+  min-width: 100%;
+  max-width: 100%;
```

Comment:
You need to add a min-height, e.g 100px, since right now a user can shrink this textarea to 20px of height